### PR TITLE
SonicWALL.NetExtender version 10.2.315

### PR DESCRIPTION
**Pull request by kevinoid**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37337)

### CODE REVIEW

File: manifests/s/SonicWALL/NetExtender/10.2.315/SonicWALL.NetExtender.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SonicWALL.NetExtender
+PackageVersion: 10.2.315
+InstallerSwitches:
+  Silent: /quiet /norestart
+  SilentWithProgress: /passive /norestart
```
Comment (Trenly) [inline manifests/s/SonicWALL/NetExtender/10.2.315/SonicWALL.NetExtender.installer.yaml:8]:
```suggestion
  SilentWithProgress: /passive /norestart
InstallerSuccessCodes: 
- 3010
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

File: manifests/s/SonicWALL/NetExtender/10.2.315/SonicWALL.NetExtender.installer.yaml (hunk 1 above)
Comment (kevinoid) [reply manifests/s/SonicWALL/NetExtender/10.2.315/SonicWALL.NetExtender.installer.yaml:8]: Unfortunately, when I add that `winget validate --manifest` fails with:

    Manifest validation failed.
    Manifest Error: Duplicate installer return code found.

I assumed 3010 was one of the ["known default values by winget"](https://github.com/microsoft/winget-cli/blob/4968df49d5ee25565fa9c37719e976c06787d97a/schemas/JSON/manifests/v1.0.0/manifest.singleton.1.0.0.json#L160).  Should I open an issue about `winget validate`?

Comment (ItzLevvie) [reply manifests/s/SonicWALL/NetExtender/10.2.315/SonicWALL.NetExtender.installer.yaml:8]: It's a bug in the latest version of WinGet: https://github.com/ItzLevvie/winget-pkgs-validate-and-install/issues/2#issuecomment-988243555

It's fixed in 1.2 and above.

Comment (kevinoid) [reply manifests/s/SonicWALL/NetExtender/10.2.315/SonicWALL.NetExtender.installer.yaml:8]: Great, thanks @ItzLevvie and @Trenly!

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**